.github/workflows/check-links.yml

```
@@ -0,0 +1,66 @@
+name: Check Documentation Links (Periodic Checks)
+
+on:
+  # Run weekdays at 10:00 UTC
+  schedule:
+    - cron: '0 10 * * 1-5'
+  # Allow manual trigger
+  workflow_dispatch:
+  # Run on PR changes to link checker config
+  pull_request:
+    paths:
+      - '.github/lychee.toml'
+      - '.github/workflows/check-links.yml'
+
+jobs:
+  links-checker:
+    # Only run on main repository for scheduled runs
+    if: ${{ github.repository == 'PecanProject/pecan' || github.event_name != 'schedule' }}
+    
+    env:
+      ISSUE_NAME: 'Documentation: broken links automatic report'
+    
+    runs-on: ubuntu-latest
+    
+    permissions:
+      issues: write  # Required for creating/updating/closing issues
+    
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0  # Fetch all history for complete link checking
+      
+      - name: Links Checker
+        id: lychee
+        uses: lycheeverse/lychee-action@v2
+        with:
+          args: --config .github/lychee.toml './**/*.md' './**/*.html' './**/*.Rd' './**/*.Rmd' './**/*.R' './**/*.qmd'
```

Including Rd will only result in duplicates since these are automatically generated.

```suggestion
          args: --config .github/lychee.toml './**/*.md' './**/*.html' './**/*.Rmd' './**/*.R' './**/*.qmd'
```